test/unit/effects.js

```diff
@@ -1028,7 +1028,7 @@ jQuery.each( {
 				jQuery( elem ).remove();
 
 			} );
-			this.clock.tick( 50 );
+			this.clock.tick( 100 );
```

Comment:
ooo, so we have animate(..., 50), but because `when.done` is now async we need value more then `50`, interesting, you probably were surprised a bit :)